apps/web-portal/index.mjs

```diff
@@ -1,9 +1,50 @@
-import {WebPortalExpressApp} from './lib'
-import {config} from './config.mjs'
+import {createLogger} from '@zorko-io/util-logger'
+import {WebPortalExpressApp, setLogger} from './lib'
+import {config} from './config'
+
+const logger = createLogger({})
+
+setLogger(logger)
 
 const app = new WebPortalExpressApp({
-  config
+  config,
 })
 
-
 app.startAndAttach()
+
+function appShutdown() {
```

Comment:
Plz, move to `WebPortalExpressApp `,  all subscription to `process` could be done in `constructor` and privat methods

Reply:
we have a stop method in `WebPortalExpressApp`. but shutdown is `process` related. do you want to handle `process` from the `WebPortalExpressApp`?

Reply:
done